test.js

```diff
@@ -204,7 +204,7 @@ test('assign styles', t => {
 });
 
 test('assign other props', t => {
-	const el = <a href="video.mp4" id="a" download referrerPolicy="no-referrer">Download</a>;
```

Comment:
I mean that in React you'd write `<img srcSet="retina.png 2x"/>`, not `<img srcset="retina.png 2x"/>`.